### PR TITLE
Lindalu fix description metadata

### PR DESCRIPTION
**Pull request by lindalu-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d6eb5fe](https://github.com/OfficeDev/office-js-docs-pr/commits/d6eb5fe4fe22fb343f916bec62de894e7a8cfffe)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-2835f666)

<a id="2835f666"></a>
###
 - **[Error]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/OfficeDev/office-js-docs-pr, working directory: W:\mhni-s, target branch: master, target branch commit: fd7dda92647e460b6987277f5eae6d8fc7beb6b0, source commit: d6eb5fe4fe22fb343f916bec62de894e7a8cfffe.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202003162132040312-1671%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=q8VUGfjt6SB1KEVx9ayAawOP%2BoWpq9433dZu9lF5TgE%3D&st=2020-03-16T21%3A27%3A19Z&se=2020-04-16T21%3A32%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by lindalu-MSFT** [comment]
@Rick-Kirkham Fixed the merge conflict, should be good to merge to master now. LMK when you're ready to publish changes!

**Comment by o365devx** [comment]
Docs Build status updates of commit _[801dbeb](https://github.com/OfficeDev/office-js-docs-pr/commits/801dbebf9f37ee70be0873b7fed61abf6849ddca)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md) | :warning:Warning | | [Details](#user-content-074bdfff-cc4b755f)
[docs/powerpoint/powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/powerpoint/powerpoint-add-ins.md) | :warning:Warning | | [Details](#user-content-071ee2d8-cc4b755f)
[docs/develop/grant-admin-consent-to-an-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/grant-admin-consent-to-an-add-in.md) | :white_check_mark:Succeeded | |
[docs/develop/loading-the-dom-and-runtime-environment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/loading-the-dom-and-runtime-environment.md) | :white_check_mark:Succeeded | |
[docs/develop/office-javascript-api-object-model.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/office-javascript-api-object-model.md) | :white_check_mark:Succeeded | |
[docs/develop/office-versions-and-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/office-versions-and-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/develop/overview-authn-authz.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/overview-authn-authz.md) | :white_check_mark:Succeeded | |
[docs/develop/persisting-add-in-state-and-settings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/persisting-add-in-state-and-settings.md) | :white_check_mark:Succeeded | |
[docs/develop/read-and-write-data-to-the-active-selection-in-a-document-or-spreadsheet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/read-and-write-data-to-the-active-selection-in-a-document-or-spreadsheet.md) | :white_check_mark:Succeeded | |
[docs/develop/register-sso-add-in-aad-v2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/register-sso-add-in-aad-v2.md) | :white_check_mark:Succeeded | |
[docs/develop/specify-office-hosts-and-api-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/specify-office-hosts-and-api-requirements.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/support-for-task-pane-and-content-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/support-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/usability-testing.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/usability-testing.md) | :white_check_mark:Succeeded | |
[docs/excel/co-authoring-in-excel-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/co-authoring-in-excel-add-ins.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-advanced-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-advanced-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-charts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-charts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-conditional-formatting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-conditional-formatting.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-data-validation.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-data-validation.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-error-handling.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-error-handling.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-multiple-ranges.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-multiple-ranges.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-ranges.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-ranges.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-shapes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-shapes.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-tables.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-tables.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-workbooks.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-workbooks.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-worksheet-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-worksheet-functions.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-worksheets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-worksheets.md) | :white_check_mark:Succeeded | |
[docs/onenote/onenote-add-ins-page-content.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/onenote/onenote-add-ins-page-content.md) | :white_check_mark:Succeeded | |
[docs/onenote/onenote-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/onenote/onenote-add-ins-programming-overview.md) | :white_check_mark:Succeeded | |
[docs/powerpoint/get-the-whole-document-from-an-add-in-for-powerpoint.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/powerpoint/get-the-whole-document-from-an-add-in-for-powerpoint.md) | :white_check_mark:Succeeded | |
[docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md) | :white_check_mark:Succeeded | |
[docs/project/create-a-project-add-in-that-uses-rest-with-an-on-premises-odata-service.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/project/create-a-project-add-in-that-uses-rest-with-an-on-premises-odata-service.md) | :white_check_mark:Succeeded | |
[docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md) | :white_check_mark:Succeeded | |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | |
[docs/publish/centralized-deployment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/publish/centralized-deployment.md) | :white_check_mark:Succeeded | |

<a id="074bdfff-cc4b755f"></a>
### [docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md)
 - **[Warning]** `````(Line: 2, Col: 102, Idx: 154) - (Line: 2, Col: 160, Idx: 212): While parsing a block mapping, did not find expected key.`````
 - **[Warning]** `````Missing attribute: title. Add a title string to show in search engine results.`````

<a id="071ee2d8-cc4b755f"></a>
### [docs/powerpoint/powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/powerpoint/powerpoint-add-ins.md)
 - **[Warning]** `````(Line: 2, Col: 95, Idx: 120) - (Line: 2, Col: 173, Idx: 198): While parsing a block mapping, did not find expected key.`````
 - **[Warning]** `````Missing attribute: title. Add a title string to show in search engine results.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202003162133297411-1671%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=GvPhmzdRLK1ff5gxbJ3Zo3Y%2FuZ%2BmapoWzoyGm9aCZSs%3D&st=2020-03-16T21%3A29%3A56Z&se=2020-04-16T21%3A34%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/powerpoint/powerpoint-add-ins.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: PowerPoint add-ins
-description: ''
+description: 'Learn how to use PowerPoint add-ins to build engaging solutions for your users' presentations across platforms including Windows, iPad, Mac, and in a browser'
```
Comment (Rick-Kirkham) [inline docs/powerpoint/powerpoint-add-ins.md]: Removing single quote inside description text.

```suggestion
description: 'Learn how to use PowerPoint add-ins to build engaging solutions for presentations across platforms including Windows, iPad, Mac, and in a browser'
```

File: docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Requesting permissions for API use in add-ins
-description: ''
+description: 'Learn about different permission levels to declare in your content or task pane add-in's manifest to specify the level of JavaScript API access.'
```
Comment (Rick-Kirkham) [inline docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md]: Remove single quote mark in description.

```suggestion
description: 'Learn about different permission levels to declare in the manifest of a content or task pane add-in to specify the level of JavaScript API access.'
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f48694c](https://github.com/OfficeDev/office-js-docs-pr/commits/f48694c0a5b1855217ce7fa3104e289db9b7b6d8)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md) | :warning:Warning | | [Details](#user-content-074bdfff-ac4d267d)
[docs/develop/grant-admin-consent-to-an-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/grant-admin-consent-to-an-add-in.md) | :white_check_mark:Succeeded | |
[docs/develop/loading-the-dom-and-runtime-environment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/loading-the-dom-and-runtime-environment.md) | :white_check_mark:Succeeded | |
[docs/develop/office-javascript-api-object-model.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/office-javascript-api-object-model.md) | :white_check_mark:Succeeded | |
[docs/develop/office-versions-and-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/office-versions-and-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/develop/overview-authn-authz.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/overview-authn-authz.md) | :white_check_mark:Succeeded | |
[docs/develop/persisting-add-in-state-and-settings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/persisting-add-in-state-and-settings.md) | :white_check_mark:Succeeded | |
[docs/develop/read-and-write-data-to-the-active-selection-in-a-document-or-spreadsheet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/read-and-write-data-to-the-active-selection-in-a-document-or-spreadsheet.md) | :white_check_mark:Succeeded | |
[docs/develop/register-sso-add-in-aad-v2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/register-sso-add-in-aad-v2.md) | :white_check_mark:Succeeded | |
[docs/develop/specify-office-hosts-and-api-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/specify-office-hosts-and-api-requirements.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/support-for-task-pane-and-content-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/support-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/usability-testing.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/usability-testing.md) | :white_check_mark:Succeeded | |
[docs/excel/co-authoring-in-excel-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/co-authoring-in-excel-add-ins.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-advanced-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-advanced-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-charts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-charts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-conditional-formatting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-conditional-formatting.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-data-validation.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-data-validation.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-error-handling.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-error-handling.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-multiple-ranges.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-multiple-ranges.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-ranges.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-ranges.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-shapes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-shapes.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-tables.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-tables.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-workbooks.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-workbooks.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-worksheet-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-worksheet-functions.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-worksheets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-worksheets.md) | :white_check_mark:Succeeded | |
[docs/onenote/onenote-add-ins-page-content.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/onenote/onenote-add-ins-page-content.md) | :white_check_mark:Succeeded | |
[docs/onenote/onenote-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/onenote/onenote-add-ins-programming-overview.md) | :white_check_mark:Succeeded | |
[docs/powerpoint/get-the-whole-document-from-an-add-in-for-powerpoint.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/powerpoint/get-the-whole-document-from-an-add-in-for-powerpoint.md) | :white_check_mark:Succeeded | |
[docs/powerpoint/powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/powerpoint/powerpoint-add-ins.md) | :white_check_mark:Succeeded | |
[docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md) | :white_check_mark:Succeeded | |
[docs/project/create-a-project-add-in-that-uses-rest-with-an-on-premises-odata-service.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/project/create-a-project-add-in-that-uses-rest-with-an-on-premises-odata-service.md) | :white_check_mark:Succeeded | |
[docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md) | :white_check_mark:Succeeded | |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | |
[docs/publish/centralized-deployment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/publish/centralized-deployment.md) | :white_check_mark:Succeeded | |

<a id="074bdfff-ac4d267d"></a>
### [docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md)
 - **[Warning]** `````(Line: 2, Col: 102, Idx: 154) - (Line: 2, Col: 160, Idx: 212): While parsing a block mapping, did not find expected key.`````
 - **[Warning]** `````Missing attribute: title. Add a title string to show in search engine results.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202003162147074111-1671%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=7XeXjOC7nTxFzBo%2FegUnI1wlAeV0jVZ%2B%2BNMasbuyxWA%3D&st=2020-03-16T21%3A43%3A31Z&se=2020-04-16T21%3A48%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d55e7ef](https://github.com/OfficeDev/office-js-docs-pr/commits/d55e7ef651295d7110410b34c67294452cb669d4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/grant-admin-consent-to-an-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/grant-admin-consent-to-an-add-in.md) | :white_check_mark:Succeeded | |
[docs/develop/loading-the-dom-and-runtime-environment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/loading-the-dom-and-runtime-environment.md) | :white_check_mark:Succeeded | |
[docs/develop/office-javascript-api-object-model.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/office-javascript-api-object-model.md) | :white_check_mark:Succeeded | |
[docs/develop/office-versions-and-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/office-versions-and-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/develop/overview-authn-authz.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/overview-authn-authz.md) | :white_check_mark:Succeeded | |
[docs/develop/persisting-add-in-state-and-settings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/persisting-add-in-state-and-settings.md) | :white_check_mark:Succeeded | |
[docs/develop/read-and-write-data-to-the-active-selection-in-a-document-or-spreadsheet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/read-and-write-data-to-the-active-selection-in-a-document-or-spreadsheet.md) | :white_check_mark:Succeeded | |
[docs/develop/register-sso-add-in-aad-v2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/register-sso-add-in-aad-v2.md) | :white_check_mark:Succeeded | |
[docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/specify-office-hosts-and-api-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/specify-office-hosts-and-api-requirements.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/support-for-task-pane-and-content-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/support-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/usability-testing.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/develop/usability-testing.md) | :white_check_mark:Succeeded | |
[docs/excel/co-authoring-in-excel-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/co-authoring-in-excel-add-ins.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-advanced-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-advanced-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-charts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-charts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-conditional-formatting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-conditional-formatting.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-data-validation.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-data-validation.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-error-handling.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-error-handling.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-multiple-ranges.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-multiple-ranges.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-ranges.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-ranges.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-shapes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-shapes.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-tables.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-tables.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-workbooks.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-workbooks.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-worksheet-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-worksheet-functions.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-worksheets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/excel/excel-add-ins-worksheets.md) | :white_check_mark:Succeeded | |
[docs/onenote/onenote-add-ins-page-content.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/onenote/onenote-add-ins-page-content.md) | :white_check_mark:Succeeded | |
[docs/onenote/onenote-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/onenote/onenote-add-ins-programming-overview.md) | :white_check_mark:Succeeded | |
[docs/powerpoint/get-the-whole-document-from-an-add-in-for-powerpoint.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/powerpoint/get-the-whole-document-from-an-add-in-for-powerpoint.md) | :white_check_mark:Succeeded | |
[docs/powerpoint/powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/powerpoint/powerpoint-add-ins.md) | :white_check_mark:Succeeded | |
[docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md) | :white_check_mark:Succeeded | |
[docs/project/create-a-project-add-in-that-uses-rest-with-an-on-premises-odata-service.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/project/create-a-project-add-in-that-uses-rest-with-an-on-premises-odata-service.md) | :white_check_mark:Succeeded | |
[docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md) | :white_check_mark:Succeeded | |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | |
[docs/publish/centralized-deployment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/lindalu-fix-description-metadata/docs/publish/centralized-deployment.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202003162147155144-1671%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=6SDLo5CjB8W%2BEq25%2FMoSUa672mEemk5LguQkjq3J1t4%3D&st=2020-03-16T21%3A44%3A55Z&se=2020-04-16T21%3A49%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**